solr/core/src/java/org/apache/solr/cloud/RecoveryStrategy.java

```diff
@@ -175,8 +175,6 @@ public final void setRecoveringAfterStartup(boolean recoveringAfterStartup) {
   }
 
   private Http2SolrClient.Builder recoverySolrClientBuilder(String baseUrl, String leaderCoreName) {
-    // workaround for SOLR-13605: get the configured timeouts & set them directly
-    // (even though getRecoveryOnlyHttpClient() already has them set)
```

Comment:
Removing the comment isn't fully what I was getting at because the code is still here pertinent to a work-around that isn't necessary.